.github/workflows/pre-submit.actions.yml

```diff
@@ -299,3 +299,52 @@ jobs:
           directory: "internal/builders/generic"
           # NOTE: compile-builder explicitly set to false.
           compile-builder: false
+
+  generate-attestations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create test inputs
+        run: |
+          cat <<EOF >DATA
+          {
+            "version": 1,
+            "attestation1.intoto":[
+              {
+              "name": "artifact11",
+              "digest": "abcde"
+              }
+            ],
+            "attestation2.intoto":[
+              {
+              "name": "artifact21",
+              "digest": "abcde"
+              }
+            ]
+          }
+          EOF
+          cat DATA > slsa-outputs.json
+
+          cat <<EOF >PREDICATE
+          {
+            "buildType": "https://github.com/slsa-framework/slsa-github-generator/go@v1",
+            "builder": {
+              "id": "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@refs/tags/v0.0.1"
+            }
+          }
+          EOF
+          cat PREDICATE > predicate.json
+      - name: Test generate attestations
+        id: generate
+        uses: asraa/slsa-github-generator/.github/actions/generate-attestations@attestation-action
```

Comment:
update asraa to this repo.

Reply:
Yes! Will do before submitting. Without this though, the pre-submits wouldn't run correct (because the action isn't checked into main -- just wanted to validate that it will go through)